vm-spot-price.py

```
@@ -1947,9 +1969,16 @@ def main() -> None:
                 "Meter",
                 "Product Name",
             ]
+            if args.estimate_monthly:
+                headers.insert(2, "$/Month")
             display_data = all_series_data
             if args.top > 0:
                 display_data = all_series_data[: args.top]
+            if args.estimate_monthly:
+                display_data = [
+                    row[:2] + [f"{float(row[1]) * 730:.2f}"] + row[2:]
```

Ruff (0.15.6)

[warning] Consider `[*row[:2], f"{float(row[1]) * 730:.2f}", *row[2:]]` instead of concatenation

Replace with `[*row[:2], f"{float(row[1]) * 730:.2f}", *row[2:]]`

(RUF005)